src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/VSCodeAnalyzerLoaderProviderFactory.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.CodeAnalysis.LanguageServer.HostWorkspace;
 
-[ExportWorkspaceService(typeof(IAnalyzerAssemblyLoaderProvider), [WorkspaceKind.Host]), Shared]
+[ExportWorkspaceService(typeof(IAnalyzerAssemblyLoaderProvider), [WorkspaceKind.Host, WorkspaceKind.MiscellaneousFiles]), Shared]
```

Comment:
Just bringing your attention to this change @dibarbet because it was added after your review. Hopefully it's okay because without it we're in a world of ALC pain and suffering :)

Reply:
seems... fine?  mostly just the default loader, except also knows about the extension context.

Reply:
Yeah, thats all it is. We need the source gen to come from the extension, because otherwise types are different in Razor when we try to actually get out generator bits, because of course all of the handlers come from the extension.